sonar-application/src/main/java/org/sonar/application/App.java

```diff
@@ -103,8 +103,22 @@ public static void main(String[] args) throws InterruptedException {
     CommandLineParser cli = new CommandLineParser();
     Properties rawProperties = cli.parseArguments(args);
 
-    App app = new App(rawProperties);
-    app.start();
+    Props props = new PropsBuilder(rawProperties, new JdbcSettings()).build();
+    ClusterProperties clusterProperties = new ClusterProperties(props);
+    clusterProperties.populateProps(props);
+
+    AppFileSystem appFileSystem = new AppFileSystem(props);
+    appFileSystem.verifyProps();
+    AppLogging logging = new AppLogging();
+    logging.configure(props);
+    clusterProperties.validate();
+
+    try (Cluster cluster = new Cluster(clusterProperties)) {
+      // used by orchestrator
+      boolean watchForHardStop = props.valueAsBoolean(ProcessProperties.ENABLE_STOP_COMMAND, false);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused "watchForHardStop" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1481)